Sudoku_Solver/Sudoku_Solver.js

```diff
@@ -0,0 +1,87 @@
+const matrix = [
+    [5, 3, 0, 0, 7, 0, 0, 0, 0],
+	[6, 0, 0, 1, 9, 5, 0, 0, 0],
+	[0, 9, 8, 0, 0, 0, 0, 0, 0],
+	[8, 0, 0, 0, 6, 0, 0, 0, 3],
+	[4, 0, 0, 8, 0, 3, 0, 0, 1],
+	[7, 0, 0, 0, 2, 0, 0, 0, 6],
+	[0, 6, 0, 0, 0, 0, 2, 8, 0],
+	[0, 0, 0, 4, 1, 9, 0, 0, 5],
+	[0, 0, 0, 0, 8, 0, 0, 7, 9]
```

Comment:
Indentation not fixed yet. Make sure you use 4 spaces instead of a tab.

Reply:
@KanchanThareja it was already fixed. Check the latest commit.

Reply:
@marwahmanbir yes it is my bad.